Routing/Loader/Reader/RestActionReader.php

```diff
@@ -236,6 +251,29 @@ public function getParents()
         return $this->parents;
     }
 
+    /**
+     * Set ignored classes.
+     *
+     * These classes will be ignored for route construction when
+     * type hinted as method argument.
+     *
+     * @param array $ignoredClasses
+     */
+    public function setIgnoredClasses(array $ignoredClasses): void
+    {
+        $this->ignoredClasses = $ignoredClasses;
+    }
+
+    /**
+     * Get ignored classes.
+     *
+     * @return array
+     */
+    public function getIgnoredClasses(): array
```

Comment:
I don't think we need a getter for this. What's the use-case?

Reply:
you might want to use it to exclude a specific class so when the default ignored classes are changed you don't have to update your code.

```php
$reader->setIgnoredClasses(arary_diff($reader->getIgnoredClasses(), [UserInterface::class]));
```

Reply:
What about adding `addIgnoredClass()`/`addIgnoredClasses()` instead then? That would also make the intent more clear IMO when reading the code that calls any of these methods.

Reply:
for this specific use case i actually need to remove an ignored class, but i agree it makes the intent a bit clearer.
do you want me to add a ``addIgnoredClass`` and ``removeIgnoredClass`` method instead of the getter and setter?

Reply:
ping @xabbuh @Tobion do you want me to change the getter / setter to add / remove?